### ISSUE TITLE
restore fail

### ISSUE THREAD

**Issue by zidingyi94** [Altinity/clickhouse-backup#248]
clickhouse-backup restore test1 --table eos_log.tb_business_log --data

error can't restore 'eos_log.tb_business_log': error during filepath.Walk for partition 'all_1_1_0': failed to crete hard link '/data/cs/clickhouse/backup/test1/shadow/eos_log/tb_business_log/default/all_1_1_0/checksums.txt' -> 'detached/all_1_1_0/checksums.txt': link /data/cs/clickhouse/backup/test1/shadow/eos_log/tb_business_log/default/all_1_1_0/checksums.txt detached/all_1_1_0/checksums.txt: file exists

clickhouse version: 21.1.6.13
clickhouse-backup version: 1.0.0
                               Build Date:      2021-06-16

What's the problem？

**Comment by Slach** [comment]
you try to restore table which already exists
try to run 
```
clickhouse-backup restore test1 --rm --table eos_log.tb_business_log
```

**Comment by AlexAkulov** [comment]
Perhaps you are writing data to this table at the same time as restoring the backup, and clickhouse creates the part `all_1_1_0` with the new data before clickhouse-backup tries to restore it.

**Comment by obazna** [comment]
@shanfeiyang if your issue is still reproducing please have a look at my fix #258. There are chances that your problem will gone.

**Comment by Slach** [comment]
let's close the issue due to inactivity
